cfpb/cfgov-refresh
# [TDP] Update TDP version to 1.1.3

- Text changes to the TDP Journey page (Changing name back to "Consumer Financial Protection Bureau").
- Fix image position on journey page
- Updates to the TDP Search page to make the Notification and Hero Image editable in Wagtail.

## Content for Editable Hero Region

### Notification text

- Currently, there are only activities for high school.
- Activities for elementary and middle school are being developed and will be available at a later time.


### Hero Image files

#### Image
![search-landing-hero-2x](https://user-images.githubusercontent.com/2914091/60348265-2e6f5f00-998d-11e9-8521-6e2f7bbb9dde.jpg)

#### Small Image
![search-landing-hero](https://user-images.githubusercontent.com/2914091/60348271-33341300-998d-11e9-9061-e2bbcded153f.jpg)


## Changes

- Replaced static content on the search page hero with editable streamfields. 

## Testing

1. Go to https://www.consumerfinance.gov/practitioner-resources/youth-financial-education/teach/activities/
2. Edit the page
3. Click the "+" below the Introduction StreamField
4. You should be able to choose "Notification" as a block option
5. In the Header Sidebar region, you should be able to add an "Image" block

## Reviewers

@Scotchester @chosak
